### PR TITLE
Changes to make copilot faster

### PR DESCRIPTION
**Pull request by Danmcaires**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -84,17 +84,17 @@ def create_vectorstore(llm):
...
                                                    messages=[{"role": "system",
-                                                              "content": "Your role is to analyze the context of a text. You should check whether the text indicates that there is information about a subject or not. If the text contains expressions like 'Im sorry', or 'no context', or 'no information', or 'i don't know', perhaps it is saying that there is not enough information, therefore the context is negative. A context will also be negative if the text says that it doesn't have access to the information."},
+                                                              "content": "Your task is to understand the context of a text. Look for clues indicating whether the text provides information about a subject. If you come across phrases such as 'I'm sorry', 'no context', 'no information', or 'I don't know', it likely means there isn't enough information available. Similarly, if the text mentions not having access to the information, or if it offers directives without the user requesting them explicitly, the context is negative."},
```
Comment (borges2157) [inline src/app.py:95]: Nice prompt!

File: src/app.py [hunk 2]
```diff
@@ -208,41 +208,32 @@ def api_response(query, session):
...
+    system_prompt = f"You are a system that choses a node in a Distributed Cloud environment. Your job is to define which of the instances given in the context, the user is asking about."
+    user_prompt = f"Make sure that only 1 is given in your response, the answer will never be more than 1 instance. If the user did not specified which instance he wants the information, you will provide the information of the instance that contains central cloud as type.\nYour answer will follow the format: {format_response}. Make sure this format is followed and nothing else is given in the your response."
 
-    output_parser = StrOutputParser()
-    chain = prompt | llm | output_parser
 
     #Get completion
-    completion = chain.invoke({"context":instance_list, "question": query})
+    completion = client.chat.completions.create(
+            model="gpt-4-turbo-preview",
```
Comment (borges2157) [inline src/app.py:223]: Is this model correct? Why?

**Review by borges2157 — COMMENTED** [review]
LGTM, I just have the doubt about the model.

File: src/app.py (hunk 2 above)
Comment (Danmcaires) [reply src/app.py:223]: It is more reliable and by my tests it didn't add more than 2 seconds of delay in response time. The key change was the response format

**Review by borges2157 — APPROVED** [review]

**Pull request merged**